interface/forms/patient_encounter/common.php

```diff
@@ -61,7 +61,7 @@ function sensitivity_compare($a, $b) {
 <head>
 <?php 
   html_header_show();
-  call_required_libraries(array("jquery-min-3-1-1","bootstrap","datepicker","fancybox"));
+  call_required_libraries(array("jquery-min-3-1-1","bootstrap","datepicker","font-awesome", "iziModalToast"));
```

Comment:
This related to setup?

Reply:
I am working to get the rest of the modals (pestering for second reviews) merged.

Reply:
dunno @aethelwulffe probably a commit from @naveen17797 not realated to setup
though i plan on using for success in ajax calls and failures top

Reply:
Or probably i messed up with my PR cause i pulled is branch for testing ??

Reply:
Dunno.
All I can say is that I 
1. create a branch off of main.
2.  Use a separate copy of the code in web directory to work and test changes.
3.  When ready to make a commit, I diff the working code against my branch code, merging them (often restricting the diff to the module directory I am working in).
4.  Review the commit to the branch.
5.  Push the commit to remote.

When there are changes to remote master that make things goofy for my working PR and I really really need to rebase everything, I typically just start a new clean branch off of origin master, then do the above diff all over again.  That is not usually an issue when I am working in a modules directory.

Reply:
Main idea above is that my repo is NOT in my web directory.  That is a generally agreed bad practice.

Reply:
copy that well said have you tested this module so far??

Reply:
why my updater commits are showing in your branch :confused:  :thinking:  @muarachmann

Reply:
Dunno surely messed up with my commit after pulling your branch for testing locally @tmccormi , @aethelwulffe can i close this and push a new commit all over???? since this  is a separate module on its own wont affect anything right